app/src/app/profile/profile.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable, BehaviorSubject } from 'rxjs';
+import { Router } from '@angular/router';
+import { User } from './user';
+import { map } from 'rxjs/operators';
+import { ProfileResponse } from './profile-response';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService {
+
+  endpoint: string = 'https://csce494.inahga.org/api';
+  user: User;
+  currUser = {};
+
+  headers = new HttpHeaders().set('Content-Type', 'application/json');
+
+  constructor(private http: HttpClient, public router: Router) { }
+
+  get isLoggedIn(): boolean {
+    let userToken = localStorage.getItme('access_token');
+    return (userToken !== null) ? true : false;
```

Comment:
```suggestion
  isLoggedIn(): boolean {
    return (localStorage.getItem('ACCESS_TOKEN') !== null;
```
No need to use the ternary operator. Also fixes a couple typos.

Reply:
Awesome! Thanks Ameer.